src/plugins/unified_search/public/__stories__/search_bar.stories.tsx

```diff
@@ -79,24 +79,10 @@ const createMockStorage = () => ({
 });
 
 const services = {
-  uiSettings: {
-    get: () => {},
-  },
-  savedObjects: action('savedObjects'),
-  notifications: action('notifications'),
-  http: {
-    basePath: {
-      prepend: () => 'http://test',
-    },
+  autocomplete: {
```

Comment:
💬 alphabetised to lower noise, and added `usageCollection` as the SearchBar component was previously expecting it was always present. Component didn't break on runtime as call to `reportUiCounter` was in an event handler